src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -64,4 +64,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyFileVersion("2.16.0.1640")]
```

Comment:
is this branch based on 2.16 release? Is it expected the nuget version number does not match the build number - I can't recall.

Reply:
The build number is based on a time dependent formula. I do not expect that to match between builds

Reply:
yes, to be clear, I meant the `2.16.0.1640` was not the same as: 
https://www.nuget.org/packages/DynamoVisualProgramming.ZeroTouchLibrary/2.16.0.2501
perhaps the final build that was published to nuget was not the same build that was committed to git.

In any case, I want to be clear I am talking about the old build number, not the new one.